lib/plugins/aws/package/compile/events/eventBridge/index.js

```diff
@@ -86,6 +86,19 @@ class AwsCompileEventBridgeEvents {
           required: ['inputTemplate'],
           additionalProperties: false,
         },
+        retryPolicy: {
+          type: 'object',
+          properties: {
+            maximumEventAge: { type: 'integer' },
```

Comment:
Are there minimum and maximum values for that property allowed by AWS? If yes, let's add them here to avoid potential CloudFormation validation issues and catch those invalid values earlier

Reply:
As per: https://docs.amazonaws.cn/en_us/eventbridge/latest/APIReference/API_RetryPolicy.html

`MaximumEventAgeInSeconds`: Minimum value of 60. Maximum value of 86400

*note*: ☝️ "InSeconds" (also used here: https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/EventBridge.html)

`MaximumRetryAttempts`: Minimum value of 0. Maximum value of 185

Reply:
And for convenience the CFN docs: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-events-rule-retrypolicy.html